src/main.cpp

```diff
@@ -729,6 +729,10 @@ static int main0(int argc, char **argv) {
                     linker_args.append(buf_create_from_str("-z"));
                     linker_args.append(buf_create_from_str(it.only_arg));
                     break;
+                case Stage2ClangArgLinkerInputL:
+                    linker_args.append(buf_create_from_str("-L"));
+                    linker_args.append(buf_create_from_str(it.only_arg));
```

Comment:
Instead of this you can simply do:

```c++
lib_dirs.append(it.only_arg);
```

This matches the `-L` parameter from zig command line which you can see earlier in the file.

Reply:
Should I leave this in or take it out?
```suggestion
            } else if (buf_eql_str(arg, "-L")) {
                i += 1;
                lib_dirs.append(buf_ptr(linker_args.at(i)));
```